scripts/pyenv_setup.sh

```diff
@@ -37,21 +37,22 @@ get_shell_startup_script() {
   echo "$_startup_script"
 }
 
-# The first \n is important on Github workers sicne it was being appended to
+# The first \n is important on Github workers since it was being appended to
 # the last line rather than on a new line. I never figured out why
 _append_to_startup_script() {
   if [[ -n "$SHELL" ]]; then
     case "$SHELL" in
     */bash)
       # shellcheck disable=SC2016
-      echo -e '\nif command -v pyenv 1>/dev/null 2>&1; then\n  eval "$(pyenv init --path)"\nfi' >>"${1}"
+      echo -e '\neval "$(pyenv init --path)"\nif command -v pyenv 1>/dev/null 2>&1; then\n  eval "$(pyenv init -)"\nfi' >>"${1}"
```

Comment:
I have not yet found someone that uses bash instead of the other other ones. The bash set up seems convoluted and I don't want to test it out.


```suggestion
      echo "Visit https://github.com/pyenv/pyenv#installation on how to fully set up your Bash shell."
```